src/functionalTest/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerControllerFunctionalTest.java

```diff
@@ -217,7 +217,7 @@ public void should_succeed_and_create_a_task_with_no_categories() {
             .body("caseTypeId.value", equalToIgnoringCase("asylum"))
             .body("jurisdiction.value", equalToIgnoringCase("ia"))
             .body("dueDate.value", notNullValue())
-            .body("taskState.value", equalToIgnoringCase("unassigned"))
+            .body("taskState.value", equalToIgnoringCase("unconfigured"))
```

Comment:
just a question, why task state expectation changed to 'unconfigured' is it related camunda-bpm?